BoxContentSDKSampleApp/BoxContentSDKSampleApp/BOXSampleAppSessionManager.m

```diff
@@ -60,6 +61,23 @@ + (NSString *)rootCacheDirGivenSharedContainerId:(NSString *)sharedContainerId
     return containerURL.path;
 }
 
++ (NSString *)tempCacheDir
```

Comment:
Not sure this method belongs in this class, it seems relatively independent of the work done by `BOXSampleAppSessionManager`